objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloQueries.java

```diff
@@ -705,6 +713,13 @@ public void testImageDateDeleteBefore() throws IOException {
         }
     }
     
+    @Test
```

Comment:
One test to fail when camera isn't online.   Will still return True when there but blank.  May have to do something fancier at some point.